gpiozero/pins/local.py

```diff
@@ -66,13 +70,39 @@ def _get_revision(self):
                     return revision
         raise PinUnknownPi('unable to locate Pi revision in /proc/cpuinfo')
 
+    @staticmethod
+    def ticks():
+        return monotonic()
+
+    @staticmethod
+    def ticks_diff(later, earlier):
+        # NOTE: technically the guarantee to always return a positive result
+        # cannot be maintained in versions where monotonic() is not available
+        # and we fall back to time(). However, in that situation we've no
+        # access to a true monotonic source, and no idea how far the clock has
+        # skipped back so this is the best we can do anyway.
```

Comment:
I wonder if it'd be worth returning `0` or `None` if `earlier > later` ?

Is the clock-skipping mentioned above referring to e.g. DST changes?

Reply:
Good idea; I'll go with 0 to keep the API simple for now.